src/site/content/en/lighthouse-performance/unminified-css/index.md

```diff
@@ -75,6 +75,34 @@ This is usually accomplished with a build tool like Gulp or Webpack.
 
 Learn how to minify your CSS code in [Minify CSS](/minify-css).
 
+## Stack-specific guidance
```

Comment:
@paulirish you originally had an AMP section here. But the link doesn't seem relevant to the topic of minification. https://github.com/GoogleChrome/web.dev/pull/4093/files#diff-ed9d24644cc976916d01f6513596a1465d8b32204949f1d5e01bef6fe536f2f8R82